source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else.jsonnet

```diff
@@ -0,0 +1,177 @@
+local rules = import '../../../lib/common_rules.libsonnet';
+local placeholders = import '../../lib/placeholders.libsonnet';
+
+local questionTitle(isPrimary) = (
+  if isPrimary then {
```

Comment:
I think it's the wrong way around now - the non primary route is showing the primary variant etc.

Reply:
I'm going to add a negative hasPrimary rule and cover these scenarios using that